frontends/web/src/api/backend.ts

```diff
@@ -77,3 +82,19 @@ export const getConfig = (): Promise<any> => {
 export const setConfig = (config: any): Promise<null> => {
   return apiPost('config', config);
 };
+
+export const signAddr = (format: string, msg: string, accountCode: AccountCode): Promise<AddrSignResp> => {
+  return apiPost('pocket-addr-sign', { format, msg, accountCode });
+};
+
+export const getPocketURL = (accountCode: string) => {
+  return (): Promise<string> => {
+    return apiGet(`exchange/pocket/buy/${accountCode}`);
+  };
+};
+
+export const isPocketSupported = (accountCode: string) => {
+  return (): Promise<boolean> => {
+    return apiGet(`exchange/pocket/supported/${accountCode}`);
+  };
+};
```

Comment:
maybe this is the right moment to start `api/exchanges.ts` and also move pocket-addr-sig in there?

Reply:
ACK